time.c

```diff
@@ -5590,89 +5611,105 @@ rb_time_zone_abbreviation(VALUE zone, VALUE time)
+ *  represent 1823-11-12 to 2116-02-20.
+ *  When Bignum or Rational is used (before 1823, after 2116, under
+ *  nanosecond), Time works slower as when integer is used.
+ *
+ *  == What's Here
+ *
+ *  \Class \Time provides methods that are useful for:
+ *
+ *  - {Creating \Time objects}[#class-Time-label-Methods+for+Creating].
+ *  - {Fetching \Time values}[#class-Time-label-Methods+for+Fetching].
+ *  - {Querying a \Time object}[#class-Time-label-Methods+for+Querying].
+ *  - {Comparing \Time objects}[#class-Time-label-Methods+for+Comparing].
+ *  - {Converting a \Time object}[#class-Time-label-Methods+for+Converting].
+ *  - {Rounding a \Time}[#class-Time-label-Methods+for+Rounding].
+ *
+ *  === Methods for Creating
+ *
+ *  - ::new: Returns a new time from specified arguments (year, month, etc.),
+ *    including an optional timezone value.
+ *  - ::local (aliased as ::mktime): Same as ::new, except the
+ *    timezone is the local timezone.
+ *  - ::utc (aliased as ::gm): Same as ::new, except the timezone is UTC.
+ *  - ::at: Returns a new time based on seconds since epoch.
+ *  - ::now: Returns a new time based on the current system time.
+ *  - #+ (plus): Returns a new time from an existing time and a positive offset.
+ *  - {.}[#method-i-2D] (minus): Returns a new time from an existing time
+ *    and a negative offset.
+ *
+ *  === Methods for Fetching
+ *
+ *  - #year: Returns the year of the time.
+ *  - #month (aliased as #mon): Returns the month of the time.
+ *  - #mday (aliased as #day): Returns the day of the month.
+ *  - #hour: Returns the hours value for the time.
+ *  - #min: Returns the minutes value for the time.
+ *  - #sec: Returns the seconds value for the time.
+ *  - #usec (aliased as #tv_usec): Returns the number of microseconds
+ *    in the subseconds value of the time.
+ *  - #nsec (aliased as #tv_nsec: Returns the number of nanoseconds
+ *    in the subsecond part of the time.
+ *  - #subsec: Returns the subseconds value for the time.
+ *  - #wday: Returns the integer weekday value of the time (0 == Sunday).
+ *  - #yday: Returns the integer yearday value of the time (1 == January 1).
+ *  - #hash: Returns the integer hash value for the time.
+ *  - #utc_offset (aliased as #gmt_offset and #gmtoff): Returns the offset
+ *    in seconds between time and UTC.
+ *  - #to_f: Returns the float number of seconds since epoch for the time.
+ *  - #to_i (aliased as #tv_sec): Returns the integer number of seconds since epoch
+ *    for the time.
+ *  - #to_r: Returns the Rational number of seconds since epoch for the time.
+ *  - #zone: Returns a string representation of the timezone of the time.
+ *
+ *  === Methods for Querying
+ *
+ *  - #utc? (aliased as #gmt?): Returns whether the time is UTC.
+ *  - #dst? (aliased as #isdst): Returns whether the time is DST (daylight saving time).
+ *  - #monday?: Returns whether the time is a Monday.
+ *  - #sunday?: Returns whether the time is a Sunday.
```

Comment:
Either `sunday?` should be before `monday?`, or after `saturday?`, or this section should be in alphabetical order.  But the current order doesn't make sense to me.

Reply:
Shifted.